draft-ietf-quic-ack-frequency.md

```diff
@@ -680,3 +680,8 @@ a permanent allocation of these frame types with the values described above.
 
 The following people directly contributed key ideas that shaped this draft:
 Bob Briscoe, Kazuho Oku, Marten Seemann.
+
+Thanks for the in-depth reviews by Lucas Pardue, Martin Thomson,
+Magnus Westerlund, Kazuho Oku, Marten Seemann, Gorry Fairhurst and
+Ingemar Johansson!
```

Comment:
```suggestion
Ingemar Johansson.
```

Nit: As much as I appreciate the feedback, I don't typically see exclamation points in IETF drafts.

Reply:
Not sure why this happened. I agree :-)